app/src/main/java/com/innowisegroup/datetimepicker/PickerDialog.kt

```diff
@@ -0,0 +1,324 @@
+        }.attach()
+
+        tabLayout?.addOnTabSelectedListener(object : TabSelectedListener {
+            override fun onTabUnselected(tab: TabLayout.Tab) {
+                datePickerFragment?.dateStub?.requestFocus()
+                timePickerFragment?.timeStub?.requestFocus()
+            }
+        })
+
+        //adds vertical divider between tabs
+        val root = tabLayout?.getChildAt(0)
+        if (root is LinearLayout) {
+            root.showDividers = LinearLayout.SHOW_DIVIDER_MIDDLE
+            val drawable = GradientDrawable()
+            drawable.setColor(ResourcesCompat.getColor(resources, android.R.color.black, null))
+            drawable.setSize(1, 1)
+            root.dividerPadding = 10
+            root.dividerDrawable = drawable
+        }
+        return view
+    }
+
+    private fun onCancelClick() {
+        dialog?.cancel()
+        cancelClickCallback?.onCancelClick()
+    }
+
+    private fun onOkClick() {
+        dialog?.dismiss()
+        if (okClickCallback != null) {
+            if (datePickerFragment == null) {
+                timePickerFragment?.timeStub?.requestFocus()
+                okClickCallback.onOkClick(
+                    LocalDateTime.of(
+                        LocalDate.now(),
+                        timePickerFragment?.localTime!!
+                    )
+                )
+                return
+            }
+            if (timePickerFragment == null) {
+                datePickerFragment?.dateStub?.requestFocus()
+                okClickCallback.onOkClick(
+                    LocalDateTime.of(
+                        datePickerFragment?.localDate!!,
+                        LocalTime.now()
+                    )
+                )
+                return
+            }
+            datePickerFragment?.dateStub?.requestFocus()
+            timePickerFragment?.timeStub?.requestFocus()
+            okClickCallback.onOkClick(
+                LocalDateTime.of(
+                    datePickerFragment?.localDate!!,
+                    timePickerFragment?.localTime!!
+                )
+            )
+        }
+    }
+
+    private fun refresh(value: String) {
+        val curItem = viewPager?.currentItem
+        tabLayout?.getTabAt(curItem!!)?.text = value
+    }
+
+    interface OkClickCallback {
+        fun onOkClick(localDateTime: LocalDateTime?)
+    }
+
+    interface CancelClickCallback {
+        fun onCancelClick()
+    }
+
+    fun showDialog(fragmentManager: FragmentManager) {
+        this.show(fragmentManager, DIALOG_TAG)
+    }
+
+    companion object {
+        private const val DIALOG_TAG = "date_time_picker_dialog"
+
+        internal const val UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"
+        internal const val UPDATE_DATE_TAB_TITLE_REQUEST_KEY = "updateDateTabTitleKey"
+
+        internal const val UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
+        internal const val UPDATE_DATE_TAB_TITLE_KEY = "updateDateTabTitleKey"
```

Comment:
You have identical string values in 2 different constants. 
UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"

To prevent potential bugs alway use different values for constants